doc/vxlan/EVPN/EVPN_VxLAN_Multihoming.md

```diff
@@ -0,0 +1,1450 @@
+- [3 Design](#3-Design)
+  - [3.1 Overview](#31-Overview)
+  - [3.2 DB Changes](#32-DB-Changes)
+    - [3.2.1 CONFIG DB](#321-CONFIG-DB)
+    - [3.2.2 APP DB](#322-APP-DB)
+    - [3.2.3 STATE DB](#323-STATE-DB)
+    - [3.2.4 ASIC DB](#324-ASIC-DB)
+    - [3.2.5 COUNTER DB](#325-COUNTER-DB)
+  - [3.3 Impact to existing modules](33-Impact-To-Existing-Modules)
+  - [3.4 SAI](#34-SAI)
+    - [3.4.1 New SAI Objects](#341-new-sai-objects)
+    - [3.4.2 Changes to Existing SAI Objects](#342-changes-to-existing-sai-objects)
+    - [3.4.3 SAI Object Usage](#343-sai-object-usage)
+  - [3.5 CLI](#35-CLI)
+    - [3.5.1 IS-CLI Compliance](#351-IS-CLI-Compliance)
+    - [3.5.2 Data Models](#352-Data-Models)
+    - [3.5.3 Configuration Commands](#353-Configuration-Commands)
+    - [3.5.4 Show Commands](#354-Show-Commands)
+    - [3.5.5 Debug Commands](#355-Debug-Commands)
+    - [3.5.6 REST API Support](#356-REST-API-Support)
+- [4 Flow Diagrams](#4-Flow-Diagrams)
+- [5 Error Handling](#5-Error-Handling)
+- [6 Serviceability and Debug](#6-Serviceability-and-Debug)
+- [7 Warm Boot Support](#7-Warm-Boot-Support)
+- [8 Scalability](#8-Scalability)
+- [9 Unit Test](#9-Unit-Test)
+  - [9.1 Functional Test Cases](#101-Functional-Test-Cases)
+  - [9.2 Negative Test Cases](#102-Negative-Test-Cases)
+  - [9.3 Warm boot Test Cases](#103-Warm-boot-Test-Cases)
+
+# List of Tables
+
+[Table 1: Abbreviations](#Table-1-abbreviations)
+
+# Revision
+
+| Rev  | Date      | Author                                           | Change Description |
+| ---- | --------- | ------------------------------------------------ | ------------------ |
+| 0.1  | 2/22/2024 | Syed Hasan Naqvi, Rajesh Sankaran, Kishore Kunal, Praveen Elagala | Initial version    |
+
+# About this Manual
+
+This document provides general information of EVPN VxLAN Multihoming feature implementation in SONiC.
+
+
+
+# Scope
+
+This document describes the high level design and changes required for supporting Multihoming in EVPN VxLAN networks.
+This feature will provide RFC 7432/8365 based Multihoming solution in SONiC.
+The ICCPd based MCLAG will continue to co-exist as an alternate solution.
+
+
+
+
+# Definition/Abbreviation
+
+## Table 1: Abbreviations
+
+| Term     | Meaning                                   |
+| -------- | ----------------------------------------- |
+| MCLAG    | Multi-Chassis LAG                         |
+| LVTEP    | Logical VTEP                              |
+| PIP, VIP | Primary IP Address, Virtual IP Address    |
+| MHD, SHD | Multi-Homed Device, Single-Homed Device   |
+| VNI      | (VxLAN) Virtual Network Identifier        |
+| L3VNI    | (VxLAN) Layer3 Virtual Network Identifier |
+| IRB VLAN | Integrated Routing and Bridging VLAN      |
+
+# 1 Feature Overview
+
+
+
+
+
+## 1.1 Requirements
+
+### 1.1.1 Functional Requirements
+
+The following are functional requirements for EVPN VxLAN Multihoming:
+
+1. Support All-EVPN based active-active access redundancy with up to 4 VTEPs.
```

Comment:
Is "up to 4 VTEPS"  just software limitation, or ASIC hardware limitaion?

Reply:
The software data-structures etc need to have some size, it is primarily for that. Not a hardware limitation.